InnerEye/ML/configs/classification/CovidHierarchicalModel.py

```diff
@@ -100,13 +102,15 @@ def get_model_train_test_dataset_splits(self, dataset_df: pd.DataFrame) -> Datas
             test_series = pd.read_csv(test_set_ids_csv).series
 
             all_series = dataset_df.series.values
-            check_all_test_series = all([series in all_series for series in test_series])
+            check_all_test_series = all(test_series.isin(all_series))
             if not check_all_test_series:
                 raise ValueError(f"Not all test series from {test_set_ids_csv} were found in the dataset.")
 
             test_set_subjects = dataset_df[dataset_df.series.isin(test_series)].subject.values
             train_and_val_series = dataset_df[~dataset_df.subject.isin(test_set_subjects)].series.values
-            num_val_samples = 400
+            random.seed(42)
+            random.shuffle(train_and_val_series)
+            num_val_samples = math.floor(0.1*len(train_and_val_series))
```

Comment:
Should this not be 10% of the full dataset length instead? Otherwise, perhaps 1/8 of `len(train_and_val_series)`?